### PR TITLE
Implement TSE corrections

### PR DESCRIPTION
**Pull request by srueda27**
Finishing up the project, a basic HTML/CSS clone of the [New York Times](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,848 @@
...
+                    <a href="#">Politics</a>
+                    <a href="#">Election 2020</a>
+                    <a href="#">New york</a>
+                    <a href="#">Bussiness</a>
+                    <a href="#">Tech</a>
+                    <a href="#">Science</a>
+                    <a href="#">Sports</a>
+                    <a href="#">Obituaries</a>
+                    <!--                                    
+                    <a href="#">Today's papers</a> -->
```
Comment (pelzolga123) [inline index.html]: Commented code should be deleted.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project needs some styling corrections, sticky footer is missing, leftmost bottom part should contain links.

**Comment by pelzolga123** [comment]
## Hi Team, your project looks :+1: great!
### But still it needs some changes: 
* [ ] Design is a little messed up in this parts, it should be fixed:
image

image

* [ ] Bottom part should be aligned like on original website:
image


Original:
image

* [ ] Bottom part should contain links to other articles with image previews on the leftmost side.

* [ ] Footer should have a fixed position (it sticks to the bottom part of the screen):
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by shshamim63** [comment]
## Status :green_apple: !! Approved !! :green_apple: 
___
Hi @srueda27 & @codecell :wave,
Well done :clap: . You have done a great job. I can see all the items are rendering as they should. Code looks neat and clean. You have used float on most of the items and used position too. In a word tremendous. 
___
#### This :+1:  is from me. you can merge it and don't forget to fill the project completion form.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](@Shakhawat)

**Pull request merged**